### PR TITLE
update gdal to 0.11

### PR DESCRIPTION
**Pull request by jdroenner**

### CODE REVIEW

**Comment by jdroenner** [comment]
[Ein neues Gdal Release erstellen mit file rename  / delete](https://trello.com/c/PI1bqS39/454-ein-neues-gdal-release-erstellen-mit-file-rename-delete)

File: operators/src/util/raster_stream_to_geotiff.rs [hunk 1]
```diff
@@ -134,7 +134,7 @@ fn gdal_writer<T: Pixel + GdalType>(
         },
     ];
 
-    let mut dataset = driver.create_with_band_type_with_options::<T>(
+    let mut dataset = driver.create_with_band_type_with_options::<T, _>(
```
Comment (ChristianBeilschmidt) [inline operators/src/util/raster_stream_to_geotiff.rs:137]: I guess we don't have to convert the path to String anymore.

File: operators/src/util/raster_stream_to_geotiff.rs [hunk 2]
```diff
@@ -47,7 +47,7 @@ where
     )
     .await?;
 
-    let bytes = gdal::vsi::get_vsi_mem_file_bytes_owned(&file_path.to_string_lossy())?;
+    let bytes = gdal::vsi::get_vsi_mem_file_bytes_owned(&*file_path.to_string_lossy())?;
```
Comment (ChristianBeilschmidt) [inline operators/src/util/raster_stream_to_geotiff.rs:50]: Same here, we can directly use the path now.

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Review by ChristianBeilschmidt — APPROVED** [review]

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Pull request merged**